lib/ui/dashboard.js

```diff
@@ -779,9 +779,49 @@ class Dashboard {
       updatePodsTable();
       screen.render();
     }
+
+    setupToggle(screen, debug);
   }
 }
 
+function setupToggle (screen, debug) {
+  const validElementValues = Object.values({
+    logs: "Logs",
```

Comment:
Can't this be a list of the actual elements, so that there is no need to lookup by label (which can be dynamically changed)?

Reply:
Hey,
maybe, you could just cycle through all elements in `screen.children` but then you will also select **Resources**, which you maybe do not want.

But the label did not change. You will see something like `Pods [your-namespace]` or `Logs [some-pod] ...` but when to take a look in `screen.children.elements[].options.label` it is still only  **Pods** or **Logs** 

I did not find an other way to identify an element, except over its label, but I'm not that firm with the blessed framework. 

Kind regards.

Reply:
Ah you're right about the `options.label`.

For the list, I was more thinking about something like:

```js
const validElementValues = [pods_table, pod_log];
```

Reply:
i did the refactoring as requested, still need to use the label to identify the element.

regarding the key binding you are right, maybe it should be unbind when leaving the view. 
The current implementation is more or less fail safe, as long as there are no elements with same labels on the other view it will work fine.

Currently I can't figure out how to un-/rebind the key in the navbar events.
Feel free to do so yourself.

Reply:
Sure, I'll look at it. Thanks.